source/manual/setting-up-new-rails-app.html.md

```diff
@@ -147,6 +148,77 @@ end
 
 Then add `task default: %i(spec lint)` in Rakefile and finally run `rake`.
 
+## Add a software licence
+
+Add a LICENCE file to the project root to specify the software licence. Unless
+your project has specific needs, you should use the [MIT License][mit-license].
+
+```
+The MIT License (MIT)
```

Comment:
Sure, I'm not too keen on linking since that is unconventional for this documentation. I've moved it to be within a details element for progressive disclosure.

Reply:
Oh yes, that's a nice compromise. Looks good 👍.